app/services/manifest_builder/manifest_service_locator_v3.rb

```diff
@@ -70,6 +70,33 @@ def canvas_builder
           iiif_annotation_page_factory: iiif_annotation_page_factory
         )
       end
+
+      def shallow_child_manifest_builder_factory
+        IIIFManifest::ManifestBuilder::ChildManifestBuilderFactory.new(
+          composite_builder: composite_builder,
+          child_manifest_builder: child_manifest_builder
+        )
+      end
+
+      def child_manifest_builders
+        composite_builder_factory.new(
+          shallow_child_manifest_builder_factory,
+          structure_builder,
+          composite_builder: composite_builder
+        )
+      end
+
+      def child_manifest_builder
+        IIIFManifest::ManifestServiceLocator::InjectedFactory.new(
+          ManifestBuilder,
+          builders: composite_builder_factory.new(
```

Comment:
Overriding this upstream in a local fork of the `iiif_manifest` Gem did successfully generate the `Ranges` for member Manifests, but they still are not rendered in the UV properly.